### PR TITLE
[ZT] Update Gateway global rules

### PR DESCRIPTION
**Pull request by maxvp**
Update global HTTP rules applied to all accounts.

### CODE REVIEW

File: content/cloudflare-one/policies/filtering/global-rules.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+# Global rules
+
+Cloudflare Zero Trust applies a set of global rules to all accounts.
+
+## DNS rules
+
+| Criteria | Value                                                                                                                                                                                             | Action | Description                                            |
+| -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------ | ------------------------------------------------------ |
+| Hostname | `*.nel.cloudflare.com`                                                                                                                                                                            | allow  | Allows L4 access to SNI domains for WARP registration. |
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/global-rules.md:16]: Should this be `*.net.cloudflare.com`?

File: content/cloudflare-one/policies/filtering/global-rules.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+| Hostname | `*.cloudflareclient.com`                                                                                                                                                                          | allow  | Allows L4 access for clients.                          |
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/global-rules.md:17]:
```suggestion
| Hostname | `*.cloudflareclient.com`                                                                                                                                                                          | allow  | Allows L4 access for the WARP client.                          |
```

File: content/cloudflare-one/policies/filtering/global-rules.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+title: Global rules
+layout: single
+weight: 6
+---
+
+# Global rules
+
+Cloudflare Zero Trust applies a set of global rules to all accounts.
+
+## DNS rules
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/global-rules.md]: What do you think about calling these sections DNS/Network/HTTP policies to match our policy builder?

File: content/cloudflare-one/policies/filtering/global-rules.md (hunk 1 above)
Comment (maxvp) [reply content/cloudflare-one/policies/filtering/global-rules.md:16]: `nel` is correct according to the source of truth.

File: content/cloudflare-one/policies/filtering/global-rules.md (hunk 3 above)
Comment (maxvp) [reply content/cloudflare-one/policies/filtering/global-rules.md]: I'm okay with that. In that case, should we change the page name to "Global policies" to match our wording elsewhere?

File: content/cloudflare-one/policies/filtering/global-rules.md (hunk 1 above)
Comment (maxvp) [reply content/cloudflare-one/policies/filtering/global-rules.md:16]: `nel` is correct according to the source of truth.

File: content/cloudflare-one/policies/filtering/global-rules.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+| Hostname | `*.nel.cloudflare.com`                                                                                                                                                                            | allow  | Allows L4 access to SNI domains for WARP registration. |
+| Hostname | `*.cloudflareclient.com`                                                                                                                                                                          | allow  | Allows L4 access for clients.                          |
+| Hostname | `*.cloudflare-gateway.com`                                                                                                                                                                        | allow  | Allows L4 access for the Gateway proxy with PAC files. |
+| Hostname | `dash.teams.cloudflare.com`, `help.teams.cloudflare.com`, `blocked.teams.cloudflare.com`, `api.cloudflare.com`, `cloudflarestatus.com`, `www.cloudflarestatus.com`, and `one.dash.cloudflare.com` | allow  | Allows L4 access for Zero Trust applications.          |
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/global-rules.md]: By Zero Trust applications, do you mean general Zero Trust features? To me, Zero Trust applications are the same thing as Cloudflare Access applications. Is there a reason these particular domains are grouped together?

File: content/cloudflare-one/policies/filtering/global-rules.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+| Hostname       | `dash.teams.cloudflare.com`                                  | bypass    | Prevents users from being locked out of the Zero Trust dashboard.                                               |
+| Hostname       | `*.dash.cloudflare.com`                                      | bypass    | Bypasses the Cloudflare dashboard and subdomains.                                                               |
+| Hostname       | `blocked.teams.cloudflare.com`                               | bypass    | Prevents an infinite loop on the Gateway block page.                                                            |
+| Hostname       | `developers.cloudflare.com` and `help.cloudflarebrowser.com` | noisolate | Prevents isolation of Cloudflare developer docs and help pages to help users troubleshoot configuration issues. |
+| Hostname       | `*.assets.browser.run`                                       | bypass    | Required for [Remote Browser Isolation (RBI)](/cloudflare-one/policies/browser-isolation/) access.              |
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/global-rules.md]:
```suggestion
| Hostname       | `*.assets.browser.run`                                       | bypass    | Required for [Remote Browser Isolation (RBI)](/cloudflare-one/policies/browser-isolation/).             |
```

File: content/cloudflare-one/policies/filtering/global-rules.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+| Hostname       | `*.edge.browser.run` and `*.cloudflarebrowser.com`           | bypass    | Required for RBI access.                                                                                        |
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/global-rules.md]:
```suggestion
| Hostname       | `*.edge.browser.run` and `*.cloudflarebrowser.com`           | bypass    | Required for RBI.                                                                                        |
```

File: content/cloudflare-one/policies/filtering/global-rules.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+| Hostname       | `*.edge.browser.run` and `*.cloudflarebrowser.com`           | isolate   | Required for RBI access.                                                                                        |
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/global-rules.md]:
```suggestion
| Hostname       | `*.edge.browser.run` and `*.cloudflarebrowser.com`           | isolate   | Required for RBI.                                                                                        |
```

File: content/cloudflare-one/policies/filtering/global-rules.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+| Hostname | `dash.teams.cloudflare.com`, `help.teams.cloudflare.com`, `blocked.teams.cloudflare.com`, `api.cloudflare.com`, `cloudflarestatus.com`, `www.cloudflarestatus.com`, and `one.dash.cloudflare.com` | allow  | Allows L4 access for Zero Trust applications.          |
+| Hostname | `*.cloudflareaccess.com`                                                                                                                                                                          | allow  | Allows L4 access for Cloudflare Access applications.   |
+
+## Proxy rules
+
+| Criteria         | Value                       | Action | Description                                                                                                                                    |
+| ---------------- | --------------------------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| Hostname         | `*.cloudflareaccess.com`    | allow  | Allows L7 security for Cloudflare Access applications.                                                                                         |
```
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/global-rules.md:26]: Not sure if "Allows L4 access" and "Allows L7 security" are clear to users.

File: content/cloudflare-one/policies/filtering/http-policies/global-rules.md
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/http-policies/global-rules.md:1]: When deleting or moving a page, be sure to add an entry to `/cloudflare-docs/content/_redirects`.

File: content/cloudflare-one/policies/filtering/global-rules.md (hunk 3 above)
Comment (ranbel) [reply content/cloudflare-one/policies/filtering/global-rules.md]: Yeah that's a good idea.

**Pull request closed without merging**